contrib/tools/meta.md/eqform.md

```diff
@@ -73,130 +86,123 @@ Currently, this list includes:
 -   Lists of formulas
 -   Sets of formulas
 
-Note that, due to preregularity requirements, the number of sorts grows
-in a multiplicative fashion every time one applies a functor to the
-formula data strcuture. For this reason, to avoid generating many unneeded
-sorts, we only generate functor liftings for:
+Note that, due to preregularity requirements, the number of sorts grows in a
+multiplicative fashion every time one applies a functor to the formula data
+strcuture. For this reason, to avoid generating many unneeded sorts, we only
+generate functor liftings for:
 
-A.  Subsorts of EqForm (12 sorts in total)
-B.  NormForm
-C.  Form
+-   Subsorts of `EqForm` (12 sorts in total)
+-   `NormForm`
+-   `Form`
 
 In practice, these are the primary cases that we care about, due to the
 considerations mentioned above.
 
 Note: many data functors (such as list and set) generate a unique sort for
 non-empty and possibly empty data structures, so that we actually generate
-2 * N + 1 additional sorts each time, where the extra sort comes from a
-base sort which helps ensure preregularity in complex cases (this gives
-29 sorts generated for both the list and set functor assuming the 14 above).
+2 * N + 1 additional sorts each time, where the extra sort comes from a base
+sort which helps ensure preregularity in complex cases (this gives 29 sorts
+generated for both the list and set functor assuming the 14 above).
 
 ```maude
 load terms.maude
 
 set include BOOL off .
 
 fmod EQFORM-IMPL{X :: TRIV} is
-  pr BOOL .
-  --- Sorts For Formulas Over Equalities
-  sort PosEqLit{X} PosEqConj{X} PosEqDisj{X} PosEqForm{X} .
-  sort NegEqLit{X} NegEqConj{X} NegEqDisj{X} NegEqForm{X} .
-  sort EqLit{X}    EqConj{X}    EqDisj{X}    EqForm{X}    .
-
-  --- Sorts For Formulas Over Everything
-  sort TrueLit{X} FalseLit{X} TruthLit{X} .
-  ---
-  sort NegTruthLit{X}  NoTrueLit{X}  NoFalseLit{X}  NormLit{X}  Lit{X}  .
-  sort NegTruthConj{X} NoTrueConj{X} NoFalseConj{X} NormConj{X} Conj{X} .
-  sort NegTruthDisj{X} NoTrueDisj{X} NoFalseDisj{X} NormDisj{X} Disj{X} .
-  sort NegTruthForm{X} NoTrueForm{X} NoFalseForm{X} NormForm{X} Form{X} .
+  pr META-TERM .
```

Comment:
I don't think we need `META-TERM` in this module, do we?

Also, we may consider using the same hooked `true` and `false` symbols as core maude by including `TRUTH-VALUE`. It would make it easier to use these formulas directly in side-conditions of rules (after the appropriate module transformation).

We can worry about that on another branch when we have an example though.

Reply:
I deleted this import.